### PR TITLE
analysis tools

### PR DESCRIPTION
**Pull request by tal-nc**
added an analysis tools folder.
should contain tools used for analysis such as israel comparison.
created there a script that gets a list of simulations folders (each created from gen_and_run_parallel) and compares them

### CODE REVIEW

File: analysis-tools/multiple_runs_comparison.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                   if
+                   (os.path.isdir(self.path / folder_name) and not folder_name == "matrix_analysis")]
+
+        self.results = []
+        for folder in folders:
+            self.results.append(pd.read_csv(folder / "final_results.csv"))
+
+    def plot_given_stats(self, stats_to_plot: List[str], ax: axis):
+        for stat in stats_to_plot:
+            means = np.mean([result[stat] for result in self.results], axis=0)
```
Comment (bgmoshe) [inline analysis-tools/multiple_runs_comparison.py]: Please also add median

File: analysis-tools/multiple_runs_comparison.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+# configuration:
+STATS_TO_COMPARE = ["was ever sick"]
+TITLE = "matrix vs vector no policy comparison"
+TO_SHOW = False
+TO_SAVE = True
+SAVE_NAME = TITLE
+
+# simulations folders
+father_folder: Path = Path(r"C:\corona\validation\matrix-vs-vector")  # optional
```
Comment (bgmoshe) [inline analysis-tools/multiple_runs_comparison.py]: *parent
Did you just assume the folder's gender???

Comment (bgmoshe) [reply analysis-tools/multiple_runs_comparison.py]: Why hardcoded? Give as input or put in a json config file

**Comment by bgmoshe** [comment]
@tal-nc  Name-wise, calling it comparing tool is incorrect. It *plots* multiple runs

File: analysis-tools/multiple_runs_comparison.py (hunk 2 above)
Comment (tal-nc) [reply analysis-tools/multiple_runs_comparison.py]: switched to parent.
i think this is small enough to have it's configuration inside of it.
this is suppose to be similar to the Israel comparison script.

**Review by bgmoshe — APPROVED** [review]

**Pull request merged**